routes/index.coffee

```diff
@@ -2,5 +2,6 @@ express = require 'express'
 router = express.Router()
 
 router.get /^\/users\/(\d+)\/(atom|rss)$/, require './users'
+router.get /^.*\/favicon.ico$/, require './favicon'
```

Comment:
OK, fair enough. So just change the regex to `/\/favicon.ico$/`, to more simply match anything ending in `/favicon.ico`.